### PR TITLE
Assignment 2

### PR DESCRIPTION
**Pull request by aparna906**
Created Program for Shape calculating Area, Perimeter and Volume.

### CODE REVIEW

File: .idea/misc.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" default="true" project-jdk-name="corretto-1.8" project-jdk-type="JavaSDK">
```
Comment (yashwantdhakad) [inline .idea/misc.xml:3]: you can add this field into gitignore

File: .idea/modules.xml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (yashwantdhakad) [inline .idea/modules.xml:1]: This file as well.

File: .idea/vcs.xml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (yashwantdhakad) [inline .idea/vcs.xml:1]: All the . files should be in gitignore

File: Corejava.iml [hunk 4]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (yashwantdhakad) [inline Corejava.iml:1]: You can add this into gitignore

File: Corejava.iml [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+<module type="JAVA_MODULE" version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="true">
+    <exclude-output />
+    <content url="file://$MODULE_DIR$">
+      <sourceFolder url="file://$MODULE_DIR$/src" isTestSource="false" />
+    </content>
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+  </component>
+</module>
```
Comment (yashwantdhakad) [inline Corejava.iml:11]: .class file should be in gitignore.

File: src/Cylinder.java [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+            return (Math.PI * radius * radius * height); //return Volume of shape Cylinder
+        } catch (ArithmeticException e) {
+            e.printStackTrace();
+            return 0.0;
+        }
+    }
+
+    public double surfaceArea() {
+        try {
+            return ((2 * 22 * radius) / 7) * (radius + height); //return Surface Area of shape Cylinder
```
Comment (yashwantdhakad) [inline src/Cylinder.java:31]: This is also good but In the above, you are using Math.PI but here you are using the value so I think you can use the same in all places.

**Review by yashwantdhakad — APPROVED** [review]
Looks good only added few minor comments.

File: src/Cylinder.java (hunk 6 above)
Comment (aparna906) [reply src/Cylinder.java:31]: Got it. I will keep this in mind.

File: .idea/misc.xml (hunk 1 above)
Comment (aparna906) [reply .idea/misc.xml:3]: Noted. Thanks.

**Pull request merged**